### PR TITLE
Update exchange-online-protection-overview.md

### PR DESCRIPTION
**Pull request by msbemba**
removed redundant information

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SecurityCompliance/eop/exchange-online-protection-overview.md [hunk 1]
```diff
@@ -43,13 +43,11 @@ EOP performs load balancing between datacenters but only within a region. If you
...
     
-- In Asia-Pacific (APAC), all Exchange Online mailboxes are located in APAC datacenters, but messages are currently routed through APAC datacenters for EOP filtering.
+- In Asia-Pacific (APAC), all Exchange Online mailboxes are located in APAC datacenters, and messages are currently routed through APAC datacenters for EOP filtering.
 
 - In the Americas, all Exchange Online mailboxes are located in U.S. datacenters, with the exception of South America where datacenters in Brazil and Chile are used and in Canada where datacenters in Canada are used. All email messages, including messages for customers in South America and Canada, are routed through local datacenters for EOP filtering; quaratined email is stored in the datacenter where the tenant is located.
     
-- In Europe, the Middle East, and Africa (EMEA), all Exchange Online mailboxes are located in EMEA datacenters, and all messages are routed through EMEA datacenters for EOP filtering.
-    
-- In Asia-Pacific (APAC), all Exchange Online mailboxes are located in APAC datacenters, and messages are currently routed through APAC datacenters for EOP filtering.
+
```
Comment (nenonix) [inline SecurityCompliance/eop/exchange-online-protection-overview.md]:
```suggestion
```

File: SecurityCompliance/eop/exchange-online-protection-overview.md [hunk 2]
```diff
@@ -43,13 +43,11 @@ EOP performs load balancing between datacenters but only within a region. If you
...
```
Comment (nenonix) [inline SecurityCompliance/eop/exchange-online-protection-overview.md]:
```suggestion
```

File: SecurityCompliance/eop/exchange-online-protection-overview.md [hunk 3]
```diff
@@ -43,13 +43,11 @@ EOP performs load balancing between datacenters but only within a region. If you
     
 - In Europe, the Middle East, and Africa (EMEA), all Exchange Online mailboxes are located in EMEA datacenters, and all messages are routed through EMEA datacenters for EOP filtering.
     
-- In Asia-Pacific (APAC), all Exchange Online mailboxes are located in APAC datacenters, but messages are currently routed through APAC datacenters for EOP filtering.
+- In Asia-Pacific (APAC), all Exchange Online mailboxes are located in APAC datacenters, and messages are currently routed through APAC datacenters for EOP filtering.
 
 - In the Americas, all Exchange Online mailboxes are located in U.S. datacenters, with the exception of South America where datacenters in Brazil and Chile are used and in Canada where datacenters in Canada are used. All email messages, including messages for customers in South America and Canada, are routed through local datacenters for EOP filtering; quaratined email is stored in the datacenter where the tenant is located.
```
Comment (nenonix) [inline SecurityCompliance/eop/exchange-online-protection-overview.md]:
```suggestion
```

**Pull request merged**